node/src/types/block.rs

```diff
@@ -1324,4 +1341,26 @@ mod tests {
             unexpected => panic!("Bad check response: {:?}", unexpected),
         }
     }
+
+    #[test]
+    fn finality_signature() {
+        let mut rng = TestRng::from_seed([2u8; 16]);
```

Comment:
Would it be better to use `TestRng::new()`, so we get a different seed every time, and the seed is printed if the test fails?

Reply:
Definitely. Thanks.

Reply:
Done in [`c0bd13f` (#683)](https://github.com/CasperLabs/casper-node/pull/683/commits/c0bd13f70e41208382667535631dd62c9a381c5c)